src/errors.rs

```diff
@@ -51,17 +54,25 @@ pub enum InfluxdbError {
     Other(String),
 }
 
-#[derive(Debug, Clone, PartialEq)]
-pub struct ServiceError(String);
+#[derive(Debug)]
+pub struct ServiceError(anyhow::Error);
 
-impl fmt::Display for ServiceError {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "ServiceError: {}", self.0)
+// Tell axum how to convert `AppError` into a response.
```

Comment:
I guess you meant `ServiceError`, not `AppError`?

Reply:
Right!